### PR TITLE
feat: Add gemini sample 

### PR DESCRIPTION
**Pull request by mario99logic**
Added sample with the simple usage of Gemini API using autokitteh connection.

### CODE REVIEW

**Comment by mario99logic** [comment]
I don't know if the requirements file is still relevant or not so I didnt add it for now.

File: samples/google/gemini/autokitteh.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# This YAML file is a declarative manifest that describes the setup
+# of an AutoKitteh project that demonstrates integration with Gemini
```
Comment (daabr) [inline samples/google/gemini/autokitteh.yaml]:
```suggestion
# of an AutoKitteh project that demonstrates integration with Gemini.
```

File: samples/google/gemini/autokitteh.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+version: v1
+
+project:
+  name: gemini_sample
+  connections:
+    - name: gemini_connection
```
Comment (daabr) [inline samples/google/gemini/autokitteh.yaml]:
```suggestion
    - name: gemini_conn
```

File: samples/google/gemini/program.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+"""
+Demonstration of AutoKitteh's Gemini integration.
+
+A single entry-point function is implemented, it sends a couple of requests to the Gemini API, and prints the responses
```
Comment (daabr) [inline samples/google/gemini/program.py:3]: "A single entry-point function is implemented" is not a description of what this module does. Keep only the rest, and make sure to use punctuation:

"Send requests to the Gemini API, and print the responses."

File: samples/google/gemini/program.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
+"""
+Demonstration of AutoKitteh's Gemini integration.
```
Comment (daabr) [inline samples/google/gemini/program.py]: Don't keep the first line of a docstring empty.
```suggestion
"""Demonstration of AutoKitteh's Gemini integration.
```

File: samples/google/gemini/program.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+from autokitteh.google import gemini_client
+
+MODEL = "gemini-1.5-flash"
+
+gemini = gemini_client("gemini_connection", model_name=MODEL)
+
+
+def on_http_get(event):
+    """Entry-point function for handling HTTP GET requests in this workflow."""
+    # Example 1: trivial interaction with Gemini
```
Comment (daabr) [inline samples/google/gemini/program.py]:
```suggestion
    # Example 1: trivial interaction with Gemini.
```

File: samples/google/gemini/program.py [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    prompt = "say meow in different languages"
+    response = gemini.generate_content(prompt)
+
+    print(response.text)
+
+    # Example 2: interactive chat using the Gemini
```
Comment (daabr) [inline samples/google/gemini/program.py]:
```suggestion
    # Example 2: interactive chat using the Gemini.
```

File: samples/google/gemini/program.py [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+gemini = gemini_client("gemini_connection", model_name=MODEL)
+
+
+def on_http_get(event):
+    """Entry-point function for handling HTTP GET requests in this workflow."""
+    # Example 1: trivial interaction with Gemini
+    prompt = "say meow in different languages"
+    response = gemini.generate_content(prompt)
+
```
Comment (daabr) [inline samples/google/gemini/program.py]: No need for this empty line. `print` is a natural part of the first 2 lines.
```suggestion
```

File: samples/google/gemini/README.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
+---
+title: Gemini
+description: Simple usage of Gemini API
```
Comment (daabr) [inline samples/google/gemini/README.md]:
```suggestion
description: Simple usage of the Gemini API
```

File: samples/google/gemini/README.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+1. Send an HTTP GET request to trigger the workflow.
+2. The workflow prints the responses of Gemini
+
+## Deployment & Configuration
+
+### Cloud Usage
+
+- Initialize your connection with Gemini through the UI
+
+### self-hosted
```
Comment (daabr) [inline samples/google/gemini/README.md]:
```suggestion
### Self-Hosted Server
```

**Review by daabr — APPROVED** [review]
Approved with various nits, please fix before merging.

File: samples/google/gemini/README.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+1. Send an HTTP GET request to trigger the workflow.
+2. The workflow prints the responses of Gemini
+
+## Deployment & Configuration
+
+### Cloud Usage
+
+- Initialize your connection with Gemini through the UI
+
```
Comment (pashafateev) [inline samples/google/gemini/README.md:31]: I think usage should include information on how to trigger as well.
I initialized the connection, now what?
We also have manual triggers now, I think that should be included as well.
Here is the most up to date [documentation for a kittehub project](https://github.com/autokitteh/kittehub/blob/main/auth0_to_hubspot/README.md)

And I think use of [github alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) makes for more dynamic documentation that's easier to read.

File: samples/google/gemini/README.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+API details:
+
+- [Gemini API](https://ai.google.dev)
+- [Python client library](https://github.com/google-gemini/generative-ai-python/blob/main/docs/api/google/generativeai.md)
+
+## How It Works
+
+1. Send an HTTP GET request to trigger the workflow.
+2. The workflow prints the responses of Gemini
```
Comment (pashafateev) [inline samples/google/gemini/README.md]:
```suggestion
1. Generate content using Gemini and log the response.
2. Conduct interactive chats with Gemini and log the responses.
```

I think what you have for number 1 in "how it works" is a better fit for "Cloud Usage" section.

**Pull request merged**